cinema-ticket/styles.css

```diff
@@ -0,0 +1,155 @@
+body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.ticket-wrapper {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  width: 780px;
+  height: 263px;
+  padding: 20px;
+  background-color: #EFC684;
+  border-radius: 10px;
+}
+
+.ticket-title {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex-basis: 27%;
+  height: 100%;
+}
+
+.ticket-title h1 {
+  margin-top: 0;
+  font-size: 55px;
+}
+
+.ticket-price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.ticket-price span {
+  padding: 10px 24px;
+  border: 1px solid #000;
+}
+
+.ticket-info {
+  flex-basis: 28%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%
+}
+
+.info-lines-wrapper {
+  margin-top: 10px;
+}
+
+.info-lines-wrapper .ticket-info-line {
+  margin-bottom: 29px;
+  border-bottom: 1px solid #000;
+  font-size: 20px;
+}
+
+.ticket-info img {
+  max-width: 100%;
+}
+
+.ticket-popcorn {
+  position: relative;
+  flex-basis: 30%;
+  display: flex;
+  justify-content: center;
+}
+
+.ticket-popcorn:before {
+  content: '';
+  position: absolute;
```

Comment:
are you sure that `position: absolute` is needed here? 
Basically position absolute is a [bad practice](https://stackoverflow.com/questions/186044/is-it-considered-bad-practice-to-use-absolute-positioning)
Try to use position relative instead, if you are having troubles with it please reach out to us on chat.